go/vt/vtctl/reparent.go

```diff
@@ -52,6 +52,12 @@ func init() {
 		commandEmergencyReparentShard,
 		"-keyspace_shard=<keyspace/shard> -new_master=<tablet alias>",
 		"Reparents the shard to the new master. Assumes the old master is dead and not responsding."})
+	addCommand("Shards", command{
+		"TabletExternallyReparented",
+		commandTabletExternallyReparented,
+		"<tablet alias>",
+		"Changes metadata in the topology server to acknowledge a shard master change performed by an external tool. See the Reparenting guide for more information:" +
+			"https://github.com/vitessio/vitess/blob/master/doc/Reparenting.md#external-reparents."})
```

Comment:
While we're at it, let's fix this old broken URL: https://vitess.io/docs/user-guides/reparenting/#external-reparenting